.github/workflows/test.yaml

```diff
@@ -0,0 +1,75 @@
+name: Test
+
+on:
+  workflow_run:
+    workflows: ["Lint and Build"]
+    types:
+      - completed
+
+jobs:
+  go:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-python@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+
+      - name: running tests
+        run: |
+          git clone https://github.com/dedis/Coding
+          ./Coding/bin/coveralls.sh
```

Comment:
```suggestion
        run: make test_goveralls
```

Reply:
That fails, because I didn't add the goveralls tokens. I'd like to get rid of goveralls and use sonarqube instead. But currently the `coveralls.sh` is the nicest script to run the `-trace -run=1 -short`...

I'll rewrite it.

Reply:
Added the simples `go test` I could think of...

Reply:
And fixed them...